Sources/PointFree/EnvVars.swift

```diff
@@ -1,8 +1,25 @@
+import Database
 import Foundation
+import GitHub
+import Mailgun
+import Stripe
+import Tagged
+
+public typealias GitHubClientId = GitHub.Client.Id
+public typealias GitHubClientSecret = GitHub.Client.Secret
+
+public typealias MailgunApiKey = Mailgun.Client.ApiKey
+public typealias MailgunDomain = Mailgun.Client.Domain
+
+public typealias StripeEndpointSecret = Stripe.Client.EndpointSecret
+public typealias StripePublishableKey = Stripe.Client.PublishableKey
+public typealias StripeSecretKey = Stripe.Client.SecretKey
```

Comment:
I think I'd even be fine dropping this type aliases. do you think they help clean things up?

Reply:
Can't unfortunately. We have `Stripe`, etc., structs inside `EnvVars` and it doesn't know how to disambiguate.

Reply:
arg. should we just go all out with `StripeClient`, `MailGunClient` and `GitHubClient`?

Reply:
yeah...I'm thinking we need a renaming PR that rethinks any potential collision :/

Reply:
yeah we can do that later, this looks good tho!